srtcore/cache.h

```diff
@@ -235,7 +236,7 @@ class CInfoBlock
 {
 public:
    uint32_t m_piIP[4];		// IP address, machine read only, not human readable format
-   int m_iIPversion;		// IP version
+   int m_iFamily;   		// IP version
```

Comment:
1. The name of the variable does not match the comment.
2. `m_iFamily` is a bit confusing to me. Maybe `m_iAddrFamily`?

Reply:
Can be also restored the previous name, I think. This was changed in order to mark all wrong uses of this field after the changes.